### ISSUE TITLE
Links are not accessible and are difficult to identify

### ISSUE THREAD

**Issue by Angelmmiguel** [bitnami/hex#194]
The [current contrast ratio](http://accessible-colors.com/) is `4.49` (#4D7BA3 / #FFFFFF, 16px) and the recommended value is `4.5`. The difference is trivial but this shows up a different issue: the links are difficult to identify in the text. We need to think how to make it more recognizable.

![screen shot 2018-07-03 at 17 45 26](https://user-images.githubusercontent.com/4056725/42230292-e8c8b6be-7ee8-11e8-8387-97f736e5bb5f.png)
